radio/src/gui/colorlcd/radio_version.cpp

```diff
@@ -360,6 +361,12 @@ void RadioVersionPage::build(FormWindow * window)
     version += options[i];
   }
 
+#if defined(RADIO_T15) && !defined(SIMU)
+  version += nl;
+  version += "PCBREV: ";
+  version += '0' + PCBREV_VALUE();
```

Comment:
does 
`version += static_cast<char>(hardwareOptions.pcbrev);`
work here (or PCBREV_VALUE() if not that as I think I only checked compile size on main version)? As it ends up a fraction smaller code-wise.

Reply:
it logically doesn't work, but `version += '0' + static_cast<char>(hardwareOptions.pcbrev); `will. Changing to that